templates/page.html

```diff
@@ -347,12 +347,12 @@ <h3>Developers</h3>
         <div class="release-news">
           <img src="{{ '/static/img/release.png'|asseturl }}" />
           <div class="release-news-body">
-            <h3>REANA 0.9.1</h3>
-            <h6>28 Sep 2023</h6>
-            <p>Visualise PDF and ROOT files in the web interface.</p>
-            <p>Authenticate users with Keycloak.</p>
-            <p>Numerous improvements to cluster performance.</p>
-            <p class="release-news-link"><a href="https://github.com/reanahub/reana/releases/tag/0.9.1">See full release notes</a></p>
+            <h3>REANA 0.9.2</h3>
+            <h6>18 Dec 2023</h6>
+            <p>Remove workflow restart limit.</p>
```

Comment:
Allow infinite number of workflow restarts.

_(To keep positive rather than negative sprint associated to the negative verb "remove".)_

Reply:
(BTW "infinite" is perhaps a tricky publicity-like word here, but it's not easy to speak about a limit of 2147483647 or some such maximum number of restarts... If we don't like "infinite", and I'm not sure I like it too, then perhaps we should say something like "Allow numerous workflow restarts" or "Allow many workflow restarts" or something like "Relieve workflow restart limit" or some such.)